conjur/controller/init_controller.py

```diff
@@ -66,25 +74,40 @@ def get_server_certificate(self):
                 # pylint: disable=raise-missing-from
                 raise RuntimeError("Error: URL is required")
 
+    # TODO: Factor out the following URL validation to ConjurrcData class
+    def format_conjur_url(self):
+        """
+        Method for formatting the Conjur server URL to
+        break down the URL into segments
+        """
         # Chops off the '/ if supplied by the user to avoid a server error
         if self.conjurrc_data.conjur_url.endswith('/'):
-            self.conjurrc_data.conjur_url=self.conjurrc_data.conjur_url[:-1]
+            self.conjurrc_data.conjur_url = self.conjurrc_data.conjur_url[:-1]
 
-        url = urlparse(self.conjurrc_data.conjur_url)
+        return urlparse(self.conjurrc_data.conjur_url)
 
-        # TODO: Factor out the following URL validation to ConjurrcData class
-        # and add integration tests
-        if url.scheme != 'https':
+    def validate_conjur_url(self, conjur_url):
+        """
+        Method for validating the Conjur server URL to
+        ensure it is provided as expected
```

Comment:
in essence renaming validate_conjur_url -> is_valid_scheme?
I don't think this will be the only validation in the future. If we think about all commands, IMO it is easier to follow a single pattern of "validation" instead of calling each validation function in each command differently. That way it is easier to read and is general enough so the reader doesnt get bogged down into the implementation details